lib/perl/Genome/Sys.pm

```diff
@@ -1522,10 +1528,15 @@ sub shellcmd {
                 {   # POE sets a handler to ignore SIG{PIPE}, that makes the
                     # pipefail option useless.
                     local $SIG{PIPE} = 'DEFAULT';
-                    $system_retval = system('bash', '-c', "$shellopts_part $cmd");
+                    my @cmdline = ('bash', '-c', "$shellopts_part $cmd");
+                    exec(@cmdline)
+                        or do {
```

Comment:
Not necessary, but it matches the pattern I try to use for these kinds of syscalls
```
function() or die '...';
```
Since there's 2 steps to "die" here, I put it in a do {} block.